plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/WriteFunction.java

```diff
@@ -20,6 +20,8 @@
 {
     Class<?> getJavaType();
 
+    int getJdbcType();
```

Comment:
This seems to be used only in the `void setNull` below.
Thus, it looks redundant. Override `setNull` instead.

Reply:
This makes this change even bigger

Reply:
I understand your concern and i also do value smaller changes. This one is about interface coherence.

Reply:
Then, I'll make setNull abstract without a default implementation. Is this ok?

Reply:
Per my understanding, we believe that the default implementation is correct, just not optimal from performance perspective. I think it warrants to be kept it as the default implementation.

Reply:
In order for oracle to work properly I need to provide correct implementation for all of the types. Thus it doesn't make sense to keep default implementation as it will be overridden.

Reply:
I am fine with making this required for Oracle.